### PR TITLE
[FT-1816] new enterprise OAS Validation plugin

### PR DESCRIPTION
**Pull request by KongSteve**
### Summary
New Validation plugin documentation

### Reason
https://konghq.atlassian.net/browse/FT-1816

### CODE REVIEW

File: app/_hub/kong-inc/oas-validation/index.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+{% navtab cURL %}
+
+```bash
+curl -X POST http://<admin-hostname>:8001/services/ \
+    --data name='Petstore-Service' \
+    --data url='https://petstore.swagger.io/v2'
+```
+
+{% endnavtab %}
+{% navtab HTTPie %}
```
Comment (mheap) [inline app/_hub/kong-inc/oas-validation/index.md:151]: We can remove HTTPie if we want to simplify

**Review by mheap — CHANGES_REQUESTED** [review]

**Comment by mheap** [comment]
Moving back to draft as it's targeted for 3.1

File: app/_hub/kong-inc/oas-validation/index.md [hunk 2]
```diff
@@ -0,0 +1,305 @@
+---
```
[the remark this answers is not in the document]
Comment (KongSteve) [reply app/_hub/kong-inc/oas-validation/index.md:1]: @lena-larionova - this plugin is targeted for 3.1. Tried the link above re icon but doesn't work. Can you let me know how we kick off the icon thing?

File: app/_hub/kong-inc/oas-validation/index.md [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+name: OAS Validation
+publisher: Kong Inc.
+version: 0.1.x
+desc: Validate HTTP requests and responses based on an OpenApi 3.0 or Swagger API Specification
```
[the remark this answers is not in the document]
Comment (vm-001) [reply app/_hub/kong-inc/oas-validation/index.md:5]: typo issue might be aware.

File: app/_hub/kong-inc/oas-validation/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (lena-larionova) [reply app/_hub/kong-inc/oas-validation/index.md:1]: Gang Guo is working with the UX team on plugin icons now. We'll get a process in place for this going forward.

**Comment by windmgc** [comment]
The OAS validation plugin document looks good to me, but I don't know why the 3.1 navigation page and mocking plugin related change are also in this PR.

@lena-larionova Can I change this PR from draft to "Ready for review"?

**Comment by lena-larionova** [comment]
> The OAS validation plugin document looks good to me, but I don't know why the 3.1 navigation page and mocking plugin related change are also in this PR.

Had a rebase issue that's now resolved, they're gone now.

> Can I change this PR from draft to "Ready for review"?

Please do.

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**